apparmor.d/groups/gnome/gnome-shell

```diff
@@ -504,7 +505,6 @@ profile gnome-shell @{exec_path} flags=(attach_disconnected) {
   /etc/fstab r,
   /etc/xdg/menus/gnome-applications.menu r,
 
-  /var/lib/gdm{3,}/.cache/ w,
```

Comment:
Put it back. It is needed to create the directory on first start.

Reply:
Sorry, mistook for my own change. There are many situations like this, I postponed them for a future PR.